components/Tabs/Tabs.js

```diff
@@ -41,18 +47,19 @@ class TabLink {
 class TabCard {
   constructor(element){
     // Assign this.element to the passed in element.
-    this.element;
+    this.element = element;
   }
   selectCard(){
     // Update the style of this.element to display = null
-    this.element;
+    this.element.style.display = null;
   }
 
 }
 
 // Create a reference to all ".tab" classes
-let tabs = document.querySelectorAll();
+let tabs = document.querySelectorAll('.tab');
 // Map over the array and convert each tab reference into a new TabLink object.  Pass in the tab object to the Tabs class.
-tabs = Array.from(tabs).map()
+tabs = Array.from(tabs).map(tab => new TabLink(tab));
```

Comment:
what's the difference between `(tab) =>` and `tab =>`?

Be consistent, whichever way you go